tests/models/whisper/test_modeling_whisper.py

```diff
@@ -1586,8 +1569,6 @@ def test_large_logits_librispeech(self):
 
     @slow
     def test_tiny_en_generation(self):
-        torch_device = "cpu"
-        set_seed(0)
```

Comment:
could you run with flakefinder to see if we need seed or not?

Reply:
`RUN_SLOW=1 py.test tests/models/whisper/test_modeling_whisper.py -k test_tiny_en_generation --flake-finder --flake-runs 100` yields no failures

(`set_seed(0)` comes from the original whisper commit. However, AFAIK, Whisper has no random components -- in fact, many output-checking tests in this file don't set a seed)